eslint.config.mjs

```diff
@@ -0,0 +1,126 @@
+import mocha from 'eslint-plugin-mocha'
+import n from 'eslint-plugin-n'
+import globals from 'globals'
+import path from 'node:path'
+import { fileURLToPath } from 'node:url'
+import js from '@eslint/js'
+import { FlatCompat } from '@eslint/eslintrc'
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+  recommendedConfig: js.configs.recommended,
+  allConfig: js.configs.all
+})
+
+export default [
+  {
+    ignores: [
+      '**/coverage',
+      '**/dist',
+      '**/docs',
+      '**/out',
+      '**/node_modules',
+      '**/versions',
+      '**/acmeair-nodejs',
+      '**/vendor',
+      'integration-tests/esbuild/out.js',
+      'integration-tests/esbuild/aws-sdk-out.js',
+      'packages/dd-trace/src/appsec/blocked_templates.js'
+    ]
+  }, ...compat.extends('eslint:recommended', 'standard', 'plugin:mocha/recommended'), {
+    plugins: {
+      mocha,
+      n
+    },
+
+    languageOptions: {
+      globals: {
+        ...globals.node,
+        ...globals.mocha,
+        sinon: false,
+        expect: false,
+        proxyquire: false,
+        withVersions: false,
+        withPeerService: false,
+        withNamingSchema: false,
+        withExports: false
+      },
+
+      ecmaVersion: 2022
+    },
+
+    settings: {
+      node: {
+        version: '>=16.0.0'
+      }
+    },
+
+    rules: {
+      'max-len': [2, 120, 2],
+      'no-var': 2,
+      'no-console': 2,
+      'prefer-const': 2,
+      'object-curly-spacing': [2, 'always'],
```

Comment:
Same as with `max-len`, this rule has been deprecated since ESLint 8.53.0. We should instead use the [corresponding rule](https://eslint.style/rules/js/object-curly-spacing) in [@stylistic/eslint-plugin-js](https://eslint.style/packages/js).